File: designer-portfolio/server/test-server.py
import http.server
import socketserver
import json

PORT = 3008

class MyHTTPRequestHandler(http.server.SimpleHTTPRequestHandler):
    def do_GET(self):
        if self.path == '/api/health':
            self.send_response(200)
            self.send_header('Content-type', 'application/json')
            self.end_headers()
            self.wfile.write(json.dumps({'status': 'ok'}).encode())
            return
        super().do_GET()

with socketserver.TCPServer(('', PORT), MyHTTPRequestHandler) as httpd:
    print(f'Server running on port {PORT}')
    httpd.serve_forever()

File: designer-portfolio/server/test-server.js
// 简单的测试服务器
import express from 'express';
const app = express();
const PORT = 3007;

app.get('/api/health', (req, res) => {
  console.log('健康检查请求');
  res.json({ status: 'ok' });
});

app.listen(PORT, '0.0.0.0', (err) => {
  if (err) {
    console.error('启动服务器错误:', err);
    return;
  }
  console.log(`Server running on http://0.0.0.0:${PORT}`);
});